pkg/csi-common/controllerserver-default.go

```diff
@@ -59,7 +59,7 @@ func (cs *DefaultControllerServer) GetCapacity(ctx context.Context, req *csi.Get
 // ControllerGetCapabilities implements the default GRPC callout.
 // Default supports all capabilities
 func (cs *DefaultControllerServer) ControllerGetCapabilities(ctx context.Context, req *csi.ControllerGetCapabilitiesRequest) (*csi.ControllerGetCapabilitiesResponse, error) {
-	glog.V(5).Infof("Using default ControllerGetCapabilities")
+	log.Tracef("Using default ControllerGetCapabilities")
```

Comment:
I don't see where we set the log level to `TraceLevel`

Reply:
changed this to debug level. We could potentially have the user define the log level using a flag instead of `debug=true| false`. 

It would be `log-level=info|debug|trace`. However I don't think we have that many logs which needs to be split into 3 levels. WDYT?

Reply:
IMO, for now, having the log level as `info|debug` should be sufficient. So `debug=true|false` should translate to `debug|info`.  maybe update all the trace level things to debug level? WDYT?

Reply:
yeah, that sounds good! Updated all trace to debug.